app/models/item.rb

```diff
@@ -35,4 +34,12 @@ def average_rating
   def apply_discount(total_items_in_cart)
     price - (price * (merchant.discounts.where("items_needed <= #{total_items_in_cart}").order(items_needed: :desc).pluck(:percentage).first * 0.01))
   end
+
+  def image_for_item
+    if self.image?
+      self.image
+    else
+      self.image = "https://images.unsplash.com/photo-1604917018135-18fe420b2ce4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
```

Comment:
In the future it may make sense to add it to the table vs. using it only for display